examples/workflow_with_custom_ops.py

```diff
@@ -0,0 +1,83 @@
+#  Copyright (c) Microsoft Corporation.
```

Comment:
Maybe `register_custom_ops.py` will be a  better name

Reply:
I think converting this script to  a test script will make it more helpful

https://github.com/microsoft/qlib/blob/main/qlib/tests/__init__.py#L9